src/aero_reps/aero_rep_single_particle.c

```diff
@@ -424,23 +431,24 @@ void aero_rep_single_particle_print(int *aero_rep_int_data,
                                     double *aero_rep_float_data) {
   int *int_data = aero_rep_int_data;
   double *float_data = aero_rep_float_data;
-}
-/*
+
   printf("\n\nSingle particle aerosol representation\n");
   printf("\nNumber of phases: %d", TOTAL_NUM_PHASES_);
   printf("\nAerosol representation id: %d", AERO_REP_ID_);
   printf("\nMax computational particles: %d", MAX_PARTICLES_);
   printf("\nParticle state size: %d", PARTICLE_STATE_SIZE_);
   printf("\n\n   - Phases -");
-  for (int i_phase = 0; i_phase < NUM_PHASE_; ++i_phase) {
-    printf("\n  state id: %d model data id: %d num Jac elements: %d",
-           PHASE_STATE_ID_(i_phase), PHASE_MODEL_DATA_ID_(i_phase),
-           PHASE_NUM_JAC_ELEM_(i_phase));
-  }
+  for(int i_layer = 0; i_layer < NUM_LAYERS_; ++i_layer){
+    for (int i_phase = 0; i_phase < NUM_PHASES_(i_layer); ++i_phase) {
+      printf("\n  state id: %d model data id: %d num Jac elements: %d",
+             PHASE_STATE_ID_(i_layer,i_phase), PHASE_MODEL_DATA_ID_(i_layer,i_phase),
+             PHASE_NUM_JAC_ELEM_(i_layer,i_phase));
+    }
```

Comment:
Do you want to modify this to output something like?:

```
  - Layers -
  layer 1
    - Phases -
      state id:...
      state id:...
  layer 2
    - Phases -
      state id:...
```

Reply:
can you clarify what you mean by this?